tools/UpdatePaths/Scripts/93676_jetpack_repaths.txt

```diff
@@ -0,0 +1,7 @@
+/obj/item/tank/jetpack/oxygen : @DELETE
```

Comment:
```suggestion
/obj/item/tank/jetpack : @DELETE
/obj/item/tank/jetpack/oxygen : /obj/item/tank/jetpack{@OLD}
```
the goal here is to turn the oxygen subtype into the base type and remove the empty type right? probably want to do this instead